stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.1.0-a
+version: 1.2.0-a
```

Comment:
@prydonius also note that I just opened https://github.com/kubernetes/charts/pull/257, which might present a merge conflict on the version number. Just something to be on the lookout for.